cmd/segment-reaper/detect.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/cfgstruct"
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/satellite/metainfo"
+)
+
+var (
+	detectCmd = &cobra.Command{
+		Use:   "detect",
+		Short: "Detects zombie segments in DB",
+		Args:  cobra.OnlyValidArgs,
+		RunE:  cmdDetect,
+	}
+
+	detectCfg struct {
+		DatabaseURL string `help:"the database connection string to use" default:"postgres://"`
```

Comment:
Can we have this flag global to the 2 commands that this tool will have?

Reply:
I prefer to leave it like that. List of global flags is already full of debug and other flags. Its also cleaner to have this flag with other flags related to this command.